internal/tar_ball_test.go

```diff
@@ -47,7 +48,9 @@ func TestS3DependentFunctions(t *testing.T) {
 
 	tarBallQueue.NewTarBall(false)
 	tarBall := tarBallQueue.LastCreatedTarball
-	tarBall.SetUp(nil)
+	if err := tarBall.SetUp(context.TODO(), nil); err != nil {
```

Comment:
Don't we want to use context.Background() instead? (But in tests, it doesn't look like it really matters.)